src/js/models/StopDetails.js

```diff
@@ -89,13 +80,18 @@ StopDetails.prototype.parseResponse = function(res) {
 
         throw new Error(faultcode + ' ' + faultstring);
     }
+    Runs = res.query.results.Envelope.Body.Nextbus2Response.Runs.Run;
```

Comment:
Sometimes Runs.Run will be a single `<Run>` element instead of a list.

Trapeze/ANUS has done this with `Trips` and `Vehicles`. See https://github.com/luqmaan/MetroRappid/blob/d418e4b422cb9e0e68e9251f1ae8935d1bc52ab3/src/js/models/VehicleCollection.js#L78-L82 and https://github.com/luqmaan/MetroRappid/blob/65e73177d341bc70a1676805733f9a7c335f3c5b/src/js/models/TripCollection.js#L18-L21